synfig-studio/src/gui/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ if(WIN32)
     configure_file(synfigstudio.rc ${CMAKE_CURRENT_BINARY_DIR}/synfigstudio.rc)
     add_executable(synfigstudio main.cpp ${CMAKE_CURRENT_BINARY_DIR}/synfigstudio.rc)
 else()
-    add_executable(synfigstudio main.cpp)
+	add_executable(synfigstudio main.cpp)
```

Comment:
indentation? I don't know what is the standard used for CMakeLists.txt :P

Reply:
I implemented my changes at first and then removed them. Saving it could have modified that indent.
CMake's also tabs :)